engine/setContext.ftl

```diff
@@ -629,8 +651,8 @@
             [#return
                 {
                     "Id" : groupId,
-                    "IsOpen" : true,
-                    "IsLocal" : false,
+                    "IsOpen" : false,
```

Comment:
This looks like a bug fix? Issue only said enhancement. Assuming we aren't currently using `_tier` anywhere otherwise it would have not worked correctly?

Reply:
Whoops will add a note for that. Just more that if you specify a Tier it shouldn't open if it can't be found and so we shouldn't open up if it can't be found